pkg/handler/eval.go

```diff
@@ -273,6 +273,17 @@ var evalSegment = func(
 			return nil, log, true
 		}
 
+		m["@entityID"] = evalContext.EntityID
+		m["@entityType"] = evalContext.EntityType
+
+		for propertyName, value := range config.Config.EvalServerEntityContext {
+			if _, ok := m[propertyName]; ok {
+				continue
+			}
+			propertyKey := fmt.Sprintf("@%s", propertyName)
+			m[propertyKey] = value
+		}
+
```

Comment:
probably the last ask is to add some documentation for this change, especially what's expected and exactly the property name people can set in their constraints.

e.g. 
```
@entityID
@entityType
@<key> from the env
```

Reply:
Yeah, I'm having a hard time figuring out where to put that documentation.  If you have any ideas?

Reply:
I think we may need to add docs at the following three places:

1. Overview doc related to the concept of "entity" and add an introduction to "entityContext". https://github.com/checkr/flagr/blame/master/docs/flagr_overview.md#L14
2. Additional description for the entityContext in evalContext https://github.com/checkr/flagr/blob/master/swagger/index.yaml#L444
3. pkg/config/env.go for a little bit more description on how the keys are transformed to `@` prefix in the entityContext

Reply:
perfect, I'll add something to those sections